src/ntrip_client/ntrip_client.py

```diff
@@ -145,26 +151,34 @@ def connect(self):
 
     # Some debugging hints about the kind of error we received
     known_error = False
+    error_code = self.ERROR_CODE_NO_ERROR
     if any(sourcetable in response for sourcetable in _SOURCETABLE_RESPONSES):
       self._logwarn('Received sourcetable response from the server. This probably means the mountpoint specified is not valid')
       known_error = True
+      error_code = self.ERROR_CODE_INVALID_MOUNTPOINT
     elif any(unauthorized in response for unauthorized in _UNAUTHORIZED_RESPONSES):
       self._logwarn('Received unauthorized response from the server. Check your username, password, and mountpoint to make sure they are correct.')
       known_error = True
+      error_code = self.ERROR_CODE_INVALID_CREDENTIALS
     elif not self._connected and (self._ntrip_version == None or self._ntrip_version == ''):
       self._logwarn('Received unknown error from the server. Note that the NTRIP version was not specified in the launch file. This is not necesarilly the cause of this error, but it may be worth checking your NTRIP casters documentation to see if the NTRIP version needs to be specified.')
       known_error = True
+      error_code = self.ERROR_CODE_UKNONW_CONNECTION_ERROR
```

Comment:
```suggestion
      error_flags = self.ERROR_FLAG_UKNOWN_CONNECTION_ERROR
```
Looks like there's a typo on this one